<launch>
  <include file="$(find track_image_contours)/launch/track_image_contours_tf.launch" />
  <node pkg="image_gui" type="DisplayMinimal.py" name="ImageDisplay" />
</launch>
